src/functions-reference/void_functions.qmd

```diff
@@ -63,3 +61,27 @@ U+000A; C++ literal `'\n'`) is inserted at the end of the printed
 line. The types `T1` through `TN` can be any of Stan's built-in
 numerical types or double quoted strings of characters (bytes).
 {{< since 2.18 >}}
+
+
+## Fatal error statement
+
+The fatal_error statement has the same syntax as the print and reject
+statements, accepting an arbitrary number of arguments of any type
+(including string literals). The effect of executing a fatal_error
+statement is to throw an exception internally that terminates the
+algorithm completely. It can be viewed as an un-recoverable version
+of reject, and as such should be used only when exiting the algorithm
+is the only option.
+
+<!-- void; fatal_error; (T1 x1,..., TN xN); -->
+\index{{\tt \bfseries fatal\_error }!{\tt (T1 x1,..., TN xN): void}|hyperpage}
+
+`void` **`fatal_error`**`(T1 x1,..., TN xN)`<br>\newline
+Print the values denoted by the arguments x1 through xN on the output
```

Comment:
x1 to XN should be escaped with back ticks to match the use of types below.

Reply:
I'm not sure what you mean, but this matches the formatting of other functions

Reply:
I meant it should be `x1` with

<pre>
`x1`
</pre>

Sorry---it's hard to write.

This is not done consistently throughout, but you can see in this example that the types are escaped this way.

Reply:
Done!